src/Persistence/ObjectManager.php

```diff
@@ -9,6 +9,8 @@
 
 /**
  * Contract for a Doctrine persistence layer ObjectManager class to implement.
+ *
+ * @method bool isUninitializedObject(mixed $value)
```

Comment:
The previous comment had a description, so let's keep it?

```suggestion
 * @method bool isUninitializedObject(mixed $value) Implementing this method will be mandatory in version 4.
```

Reply:
updated thanks